tasks/clientlibs.js

```diff
@@ -254,16 +259,18 @@ module.exports = function (grunt) {
 							clientLibJS = getIncludeString(clientLibName, 'js');
 
 							// Sort the array into dependency order
-							clientLibJSObj = performSort(clientLibJSObj);
+							clientLibJSOrderObj = performSort(clientLibJSObj);
```

Comment:
Same as comment on the clientLibCSSOrderObj variable change from clientLibCSSObj

Reply:
same with above